addons/website_event/views/assets.xml

```diff
@@ -9,7 +9,7 @@
 
     <template id="assets_tests" name="Event Assets Tests" inherit_id="web.assets_tests">
         <xpath expr="." position="inside">
-            <script type="text/javascript" src="/website_event/static/src/js/tours/website_event.js"></script>
+            <script type="text/javascript" src="/website_event/static/tests/tours/website_event.js"></script>
```

Comment:
About [MOV] website_event: move the website_event tour to tests/tours] Couldn't it be merged with the other commit about tour change ? Seems like we could gain one commit :) .

Reply:
No problem, I was not sure I could do it, because in the guidelines it is said that I should not change the content of the moved file

Reply:
Be a pirate :pirate_flag: